docs/guides/hanadb/custom-rbac/using-custom-rbac.md

```
@@ -0,0 +1,230 @@
+title: Run HanaDB with Custom RBAC Resources
+menu:
+  docs_{{ .version }}:
+    identifier: hanadb-custom-rbac-quickstart
+    name: Custom RBAC
+    parent: hanadb-custom-rbac
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)